config_defaults_inc.php

```diff
@@ -2697,7 +2697,17 @@
 
 /**
  * access level needed to be able to be listed in the assign to field.
- * @global integer $g_handle_bug_threshold
+ * if a single value provided, this means that all users that have 
+ * access level >= this value, will be listed.
```

Comment:
- It's a proper sentence, so start with capital letter.
- Remove _this means that_
- Missing _**is** provided_ 
- Remove comma before _will be listed_